### DISCUSSION TITLE
Limited views for non logged in users?

### DISCUSSION THREAD

**Discussion by maco1717** [fisharebest/webtrees#4831] [General]
Hi I was trying to figure out how can I lock the page down to only have some basic HTML information, no menu and a login option.

Is there a way to achieve this?

Mainly the menu. How do I make it disappear for non-logged in users? I this configurable via web ui or do I need to start changing code?

I thought about a splash screen in front as a proxy this is a possibility with some basic auth. Ideally I would much rather the application to support this. If it doesn't already.

Thanks,
\M

**Comment by maco1717** [comment]
I think I figured it out. Basically change everything in the family tree privacy settings to members.

The one I cannot understand it the show private relationships. there is only the options of visitors and hide from everyone. Surely there should be one or two more options here.

\M

**Comment by fisharebest** [comment]
> The one I cannot understand it the show private relationships.

Internally, this is a yes/no configuration option.

We express it as "visitors and hide from everyone" for a more consistent UI.
